screens/CheckoutScreen.js

```diff
@@ -231,14 +191,14 @@ export default class CheckoutScreen extends React.Component {
                 ))}
               </ScrollView>
             </View>
-            <View style={{ height: '20%', paddingBottom: '5%' }}>
-              <TextHeader>Rewards Applied</TextHeader>
-              <ScrollView style={{ alignSelf: 'flex-end' }}>{this.generateRewardsApplied()}</ScrollView>
-            </View>
-            <View style={{ height: '20%', paddingBottom: '5%' }}>
-              <TextHeader>Rewards Available</TextHeader>
-              <ScrollView style={{ alignSelf: 'flex-end' }}>{this.generateRewardsAvailable()}</ScrollView>
-            </View>
+            {/* Should be greyed out if totalPrice < 5 */}
```

Comment:
I think we talked about this earlier but will we ever have a situation where customers want to use a reward on a transaction less than $5 so they don't have to pay any money? Probs not in scope for this PR but something to think about...we'd also have to consider what this means in terms of logic for having the button grayed out (i.e. changing it to a clickable button if a user has rewards in their account at all vs only having it be clickable when they have rewards and the transaction > 5)

Reply:
@12aschen another design question for the future xd

Reply:
Have added this to quacestions doc!